nnsuite/pytorch
# [Packaging] Use protobuf v3.8

This patch uses protobuf v3.8 instead of v3.6.1.

Signed-off-by: Sangjung Woo <sangjung.woo@samsung.com>


### Known issue
After applying this patch, Unit testcase `test_tensor_filter.framework_auto_ext_pt_01` of NNStreamer is failed.
* Test envrionment
  * Apply this patch to pytorch, build and install pytorch debian package
  * Disable `tensorflow` and `tensorflow-lite` and Enable `pytorch` and `caffe2` when building NNStreamer since Tensorflow also has protobuf version problem.
  * Build NNStreamer and run unit tests

* Test result
```bash
...

[ RUN      ] test_tensor_filter.framework_auto_ext_pt_01
** Message: 08:23:25.262: gpu = 0, accl = default
[libprotobuf FATAL ../third_party/protobuf/src/google/protobuf/generated_message_reflection.cc:2321] CHECK failed: file != NULL: 
unknown file: Failure
C++ exception with description "CHECK failed: file != NULL: " thrown in the test body.
[  FAILED  ] test_tensor_filter.framework_auto_ext_pt_01 (24 ms)
[----------] 3 tests from test_tensor_filter (137 ms total)

[----------] Global test environment tear-down
[==========] 61 tests from 3 test cases ran. (156 ms total)
[  PASSED  ] 60 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] test_tensor_filter.framework_auto_ext_pt_01

```
